### ISSUE TITLE
Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by laleft** [sebhildebrandt/systeminformation#438]
**Describe the bug**
Using `processLoad `in observer throws `TypeError: Cannot read property 'replace' of undefined`

**To Reproduce**
```
const si = require('systeminformation');

// define all values, you want to get back
valueObject = {
  processLoad: 'mysql,postgres'
}

function usersCallback(data) {
  console.log(data);
}

// now define the observer function
let observer = si.observe(valueObject, 1000, usersCallback);

// In this example we stop our observer function after 30 seconds
setTimeout(() => {
  clearInterval(observer)
}, 30000);
```

**Current Output**
```
PS D:\draft\moni> node index.js
D:\draft\moni\node_modules\systeminformation\lib\util.js:494
  result = result.replace(/>/g, '');
                  ^

TypeError: Cannot read property 'replace' of undefined
    at Object.sanitizeShellString (D:\draft\moni\node_modules\systeminformation\lib\util.js:494:19)
    at process.nextTick (D:\draft\moni\node_modules\systeminformation\lib\processes.js:840:34)
    at process._tickCallback (internal/process/next_tick.js:61:11)
PS D:\draft\moni> node index.js
D:\draft\moni\node_modules\systeminformation\lib\util.js:494
  result = result.replace(/>/g, '');
                  ^

TypeError: Cannot read property 'replace' of undefined
    at Object.sanitizeShellString (D:\draft\moni\node_modules\systeminformation\lib\util.js:494:19)
    at process.nextTick (D:\draft\moni\node_modules\systeminformation\lib\processes.js:840:34)
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

**Expected behavior**
```
[ { name: 'mysql',
    running: true,
    startmode: '',
    pids: [ 4126, 4292 ],
    pcpu: 0.25911441952494146,
    pmem: 3.6 },
  { name: 'postgres',
    running: false,
    startmode: '',
    pids: [],
    pcpu: 0,
    pmem: 0 } ]
```

**Environment (please complete the following information):**
- node v10.16.0
- OS: CentOS 7 and Windows 10 (same result)

**Comment by sebhildebrandt** [comment]
@laleft thank you for your report. There are several things to say here:

1. `processLoad` currently expects only ONE process name. But it might be a good idea to extend this. Not now, because this would also change the result type of this function what would be a breaking change. I will keep this probably for the upcoming version 5
2. in your value object you are in the VALUE part of the `processLoad`, you are trying to provide `parameters` to this function, but this is not how this intended to be. The value part should have the names of the the KEYS you need from the result object. 
3. So at the moment we do not have a possibility to provide parameters to the observes function. But this is something I will implement. 

So this should work, as soon as I have implemented the possibility to provide parameters. I am thinking of doing it like so:

```
const si = require('systeminformation');

// define all values, you want to get back
valueObject = {
  // where proc and cpu are the result keys you are interested in, 
  // postgres is the parameter you would like to pass to the processLoad function
  processLoad: 'proc, cpu (postgres)'   
}

function usersCallback(data) {
  console.log(data);
}

// now define the observer function
let observer = si.observe(valueObject, 1000, usersCallback);

// In this example we stop our observer function after 30 seconds
setTimeout(() => {
  clearInterval(observer)
}, 30000);
```

I'll inform you as soon as I have implemented a possibility to also pass parameters within the valueObject.

**Comment by sebhildebrandt** [comment]
@laleft I now extended the `get()` function to accept parameters in the valueObject. The documentation for the `get()` function can be found here: https://systeminformation.io/general.html This allows you to also provide parameters in the `observe()` value object. The example I gave you in my previous comment should now work. I also cleaned up some things so that the catching of errors is now better.

Remember, the `processLoad()` function still accepts only ONE process as a parameter. 

Version 4.30.0 just published.
